crates/api/src/builder/api.rs

```diff
@@ -177,10 +179,10 @@ where
                     .map(|data| data.signed_constraints)
                     .collect::<Vec<SignedConstraints>>();
         
-                Ok(Json(constraints))
+                Ok(Json(constraints).into_response())
             }
             Ok(None) => {
-                Ok(Json(vec![])) // Return an empty vector if no constraints are found
+                Ok(StatusCode::NO_CONTENT.into_response())
```

Comment:
can you revert this back to an empty array? following the latest update in https://github.com/chainbound/bolt-docs/pull/15#discussion_r1777422621

same for delegations endpoint :D

Reply:
resolved